endpoints/base

```diff
@@ -388,33 +388,6 @@ function do_roadblock() {
     fi
 }
 
-function process_prebench_roadblocks() {
-    do_roadblock "endpoint-deploy" "follower" $endpoint_deploy_timeout
-    if [ $? -eq $rb_exit_abort ]; then
-        exit_error "Exiting due to abort from another participant"
-    fi
-    do_roadblock "client-server-script-start" "follower" $client_server_script_start_timeout
-    if [ $? -eq $rb_exit_abort ]; then
-        exit_error "Exiting due to abort from another participant"
-    fi
-    do_roadblock "client-server-get-data" "follower" $default_timeout
-    if [ $? -eq $rb_exit_abort ]; then
-        exit_error "Exiting due to abort from another participant"
-    fi
-    do_roadblock "client-server-collect-sysinfo" "follower" $default_timeout
-    if [ $? -eq $rb_exit_abort ]; then
-        exit_error "Exiting due to abort from another participant"
-    fi
-    do_roadblock "client-server-start-tools" "follower" $default_timeout
-    if [ $? -eq $rb_exit_abort ]; then
-        exit_error "Exiting due to abort from another participant"
-    fi
-    do_roadblock "client-server-start-tests" "follower" $default_timeout
-    if [ $? -eq $rb_exit_abort ]; then
-        exit_error "Exiting due to abort from another participant"
-    fi
-}
-
```

Comment:
Sort of.  I determined that these were blocks of code that were only used by the localhost endpoint and would have been orphaned if they had left behind.

It is important to remember that @atheurer wrote the localhost endpoint as a means of doing the initial bootstrap of rickshaw and that is has been completely idle and abandoned ever since.  As such, all of that code has been completely unmaintained for awhile and left to rot...these functions are a perfect example of something that is no longer being called by the actively used and maintained endpoints.